### PR TITLE
Lvnodj 164/view posts route

### PR DESCRIPTION
**Pull request by didOstap**

### CODE REVIEW

File: client/src/components/Main/Feed.tsx [hunk 1]
```diff
@@ -1,19 +1,50 @@
...
+    feed: IState;
+}
+
+interface IProps {
+    username: string;
+    getUserInfoFromToken: () => void;
+}
+
+class Feed extends React.Component<IProps> {
+    public state: any = {
```
Comment (InnaKutsik) [inline client/src/components/Main/Feed.tsx]: Any?

File: client/src/store/login/actions.ts [hunk 2]
```diff
@@ -30,9 +30,8 @@ export const loginUser = (user: IUser): (dispatch: Dispatch) => Promise<void> =>
             dispatch(getUserPending());
             const token = localStorage.getItem(TOKEN);
             const get = await API.get(('/'), {
-                headers: {Authorization: token},
+                headers: {'x-access-token': token},
```
Comment (InnaKutsik) [inline client/src/store/login/actions.ts]: Pls move to const

File: client/src/components/Main/Feed.tsx (hunk 1 above)
Comment (didOstap) [reply client/src/components/Main/Feed.tsx]: fixed

File: client/src/store/login/actions.ts (hunk 2 above)
Comment (didOstap) [reply client/src/store/login/actions.ts]: fixed

**Pull request merged**